.gitignore

```diff
@@ -1,13 +1,50 @@
-*.log
-*.pot
 *.pyc
+*.swp
+*.lock
+*.log
+*.pid
+cms/django
+*.DS_Store
+*.svn
+.*
+!.travis.yml
+!.coveragerc
+!.codeclimate.yml
```

Comment:
@Chive this is basically to make sure that .travis.yml is required (it negates a given pattern) though I have no idea why it's done like this here. I copied from aldryn-newsblog